### PR TITLE
PMM-12118 Add support for the official clickhouse plugin

### PR DESCRIPTION
**Pull request by 0leksii**
Do not strip binaries while building the percona-dashboards RPMs
to avoid checksum inconsistencies for grafana plugins. Add new
options to pass to grafana for setting up the new datasource.
Setup new datasource for Grafana.

PMM-12118

Link to the Feature Build: SUBMODULES-3269

Related PRs:
- https://github.com/percona/grafana-dashboards/pull/1503

### CODE REVIEW

**Comment by BupycHuk** [comment]
Shouldn't we remove the old one?

File: update/ansible/playbook/tasks/files/datasources.yml [hunk 1]
```diff
@@ -48,3 +48,9 @@ datasources:
   url: http://localhost:9093/alertmanager/
   jsonData:
     keepCookies: []
+- name: ClickHouse (New)
+  type: grafana-clickhouse-datasource
+  jsonData:
+    port: ${PERCONA_TEST_PMM_CLICKHOUSE_PORT}
+    server: ${PERCONA_TEST_PMM_CLICKHOUSE_HOST}
+    tlsSkipVerify: true
```
Comment (atymchuk) [inline update/ansible/playbook/tasks/files/datasources.yml:56]: What if the database is provisioned externally? It seems like we may need TLS.
@0leksii @BupycHuk What do you think?

**Comment by atymchuk** [comment]
> Shouldn't we remove the old one?

I also think we should.

**Comment by 0leksii** [comment]
> Shouldn't we remove the old one?

My understanding was that if it is in use by any of the dashboards - we cannot simply remove it, but deprecate and remove in 5 releases. 
From what I see, we use an old plugin in `Environments Overview (Designed for PMM)` dashboard. This one is in experimental, so should we consider the plugin in use, or for experimental dashboards we can ignore that?
@rnovikovP ^^ ??

**Comment by 0leksii** [comment]
> The main purpose of this task was to get rid of CVE. And not removing the old plugin doesn't fix the main problem. In that case we need to double an effort and update the old plugin as well.

I agree, but updating the plugin to the latest version won't help as it still have reported CVEs. You can check in comments here: https://jira.percona.com/browse/PMM-12118

**Comment by atymchuk** [comment]
> > The main purpose of this task was to get rid of CVE. And not removing the old plugin doesn't fix the main problem. In that case we need to double an effort and update the old plugin as well.
> 
> I agree, but updating the plugin to the latest version won't help as it still have reported CVEs. You can check in comments here: https://jira.percona.com/browse/PMM-12118

This is true. However, native Grafana plugins usually run a higher chance of getting CVEs fixes asap, compared to plugins contributed by the community. We need to remember that our codebase is a few minor versions behind upstream, where those CVEs may have already been fixed, especially the critical and high severity ones.

File: update/ansible/playbook/tasks/files/datasources.yml (hunk 1 above)
Comment (0leksii) [reply update/ansible/playbook/tasks/files/datasources.yml:48]: Done

Comment (atymchuk) [reply update/ansible/playbook/tasks/files/datasources.yml:48]: > Done

I'm curious: how did we address it?

File: update/ansible/playbook/tasks/files/datasources.yml [hunk 2]
```diff
@@ -48,3 +40,9 @@ datasources:
   url: http://localhost:9093/alertmanager/
   jsonData:
     keepCookies: []
+- name: ClickHouse
```
Comment (atymchuk) [inline update/ansible/playbook/tasks/files/datasources.yml:43]: I believe we need to delete the previous one by inserting these lines at the very top:
```
deleteDatasources:
  - name: ClickHouse
    orgId: 1
```

File: update/ansible/playbook/tasks/files/datasources.yml [hunk 3]
```diff
@@ -48,3 +40,9 @@ datasources:
...
+  type: grafana-clickhouse-datasource
```
Comment (atymchuk) [inline update/ansible/playbook/tasks/files/datasources.yml:44]: Can we add `orgId: 1` here as well?

File: update/ansible/playbook/tasks/files/datasources.yml (hunk 1 above)
Comment (0leksii) [reply update/ansible/playbook/tasks/files/datasources.yml]: Ouch.... It seems that I disabled it. And then thought I forgot to switch the flag and switched it back....
Sorry for that. Will disable it.

**Review by ritbl — APPROVED** [review]

**Review by atymchuk — APPROVED** [review]

**Pull request merged**